### PR TITLE
Added Popup detail window

### PR DESCRIPTION
**Pull request by DyaryRaoof**
Added few image files
Added pop-up window to desktop and mobile
Added javascript to make the popup window functional
Added a relevant javascript object to different modals when the `see-project` button is pressed for each work card.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -123,7 +123,8 @@ <h1>Tonic</h1>
                             <li>css</li>
```
Comment (carlos-ssh) [inline index.html:123]: You are missing to add your linters for JS.
- Kindly add the required[ Linters](https://github.com/microverseinc/linters-config/tree/master/javascript) for this activity.

File: script.js [hunk 2]
```diff
@@ -4,6 +4,86 @@ const closeButton = document.querySelector('#close');
...
+  name: 'Uber Navigation',
+  languages: ['Uber', 'Lead Developer', '2018'],
+  description:
+    'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+  image: 'images/fourth-work-card-desktop.png',
+  technologies: ['html', 'Ruby on Rails', 'css', 'javascript'],
+  liveVersion: '',
+  linkToSource: '',
+};
+
```
Comment (carlos-ssh) [inline script.js]: The JavaScript code needs to implement an array of objects to store JavaScript objects containing the information for each project.

- Kindly work on these changes:

Screen Shot 2021-08-31 at 19 49 56

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @DyaryRaoof , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 2 above)
Comment (DyaryRaoof) [reply script.js]: Hi. Thank you for the review. I put object1, object2, object3, and object4 into an array just below that code. in the  `cardObjectsArray ` should make changes to that?

Comment (carlos-ssh) [reply script.js]: I saw your changes. Thank you :D

**Review by carlos-ssh — APPROVED** [review]
Hi @DyaryRaoof , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**